### ISSUE TITLE
Mixxx ends with SIGSEGV 

### ISSUE THREAD

**Issue by Holzhaus** [Holzhaus/mixxx-gh-issue-migration#167]
This was tested with build 1.12-r5443 on Windows XP 

[Thread 1400.0xbf8 exited with code 0]

Program received signal SIGSEGV, Segmentation fault.
0xfeeefeee in ?? ()
(gdb) bt
#0  0xfeeefeee in ?? ()
#1  0x10bc65c0 in ?? ()
#2  0x10d14fc1 in ?? ()
#3  0x10a62527 in ?? ()
#4  0x10a724d4 in ?? ()
#5  0x10a7257c in ?? ()
#6  0x7c91118a in ntdll!LdrInitializeThunk ()
   from C:\WINDOWS\system32\ntdll.dll
#7  0x10a60000 in ?? ()
#8  0x7c9324ca in ntdll!RtlDestroyEnvironment ()
   from C:\WINDOWS\system32\ntdll.dll
#9  0x7c81bf3e in KERNEL32!IsValidLocale ()
   from C:\WINDOWS\system32\kernel32.dll
#10 0x7c81bfb6 in KERNEL32!ExitProcess ()
   from C:\WINDOWS\system32\kernel32.dll
#11 0x00a43fac in MSVCR120!?_query_new_mode@@YAHXZ ()
   from C:\WINDOWS\system32\msvcr120.dll
#12 0x00000000 in ?? ()
(gdb)

**Comment by Holzhaus** [comment]
(by ywwg)
has this happened again?  this backtrace is not very helpful so this is impossible to diagnose

**Comment by Holzhaus** [comment]
(by daschuer)
I do not use windows regularly.
I will try to reproduce it again. Maybe it is the known issue with direct connections in  COT.
Would you mind to have a look to 
https://github.com/mixxxdj/mixxx/pull/713
which contain some easy changes towards a complete COT solution?

**Comment by Holzhaus** [comment]
(by daschuer)
I have just reproduces it. After starting Mixxx 1.12-r5553 7 times. 

"Unbehandelte Ausnahme bei 0x00000000 in mixxx.exe: 0xC0000005: Zugriffsverletzung beim Lesen an Position 0x00000000."

Log: 

Unbehandelte Ausnahme bei 0x00000000 in mixxx.exe: 0xC0000005: Zugriffsverletzung beim Lesen an Position 0x00000000.


00000000()	
mixxx.exe!0065f8c6() 	
[Unten angegebene Rahmen sind möglicherweise nicht korrekt und/oder fehlen, keine Symbole geladen für mixxx.exe]	
msvcr120.dll!00a2ecfa()

**Comment by Holzhaus** [comment]
(by ywwg)
I guess this bug is de facto blocked by the lack of debug files on our windows builds

**Comment by Holzhaus** [comment]
(by ywwg)
(blocked by lp/1456107)

**Comment by Holzhaus** [comment]
(by daschuer)
I have just reproduces it with Mixxx 2.0.0 (build 1.12 r5772) on Ubuntu Precise 
and it was reported here:  Bug #1495317

**Comment by Holzhaus** [comment]
(by daschuer)
This old big is no longer relevant.
